.github/workflows/ci.yml

```diff
@@ -17,27 +17,12 @@ jobs:
     with:
       toolchain: stable
       publish: false
-  wasm:
```

Comment:
Is there any risk to discover later (in project where `crypto_core` is required) than `crypto_core` is not wasm32 compliant?

Reply:
Indeed, needed to  keep the specification of the `getrandom` feature in the `Cargo.toml`